app/src/main/java/mozilla/lockbox/adapter/ItemListAdapter.kt

```diff
@@ -42,21 +45,30 @@ class ItemListAdapter : RecyclerView.Adapter<ItemListCell>() {
             NO_MATCHING_ENTRIES_CELL_TYPE -> {
                 val view = inflater.inflate(R.layout.list_cell_no_matching, parent, false)
 
+                view.noMatchingLearnMore
+                    .clicks()
+                    .subscribe(noMatchingEntriesClicks as PublishSubject)
```

Comment:
Why is `noMatchingEntriesClicks` using `as PublishSubject`, but ln 59 and ln 71 are using `as Subject`? Can we make them consistent?

Reply:
yesssssssssssssssssssssssssss